<?xml version="1.0"?>
<launch>
    <node name="pressure_converter" pkg="bitbots_ros_control" type="pressure_converter">
        <rosparam command="load" file="$(find bitbots_ros_control)/config/pressure_converter.yaml" />
        <rosparam command="load" file="$(find bitbots_ros_control)/config/pressure_$(optenv ROBOT_NAME nobot).yaml" />
    </node>

</launch>
